packages/components/psammead-image/package.json

```diff
@@ -16,14 +16,16 @@
     "url": "https://github.com/bbc/psammead/issues"
   },
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image/README.md",
-  "dependencies": {
-    "prop-types": "^15.6.2",
-    "styled-components": "^4.1.2"
-  },
   "devDependencies": {
     "@bbc/psammead-assets": "^0.1.3",
     "@bbc/psammead-test-helpers": "^0.3.0",
-    "react": "^16.6.3"
+    "react": "^16.6.3",
+    "prop-types": "^15.6.2",
+    "styled-components": "^4.1.3"
+  },
+  "peerDependencies": {
+    "prop-types": "^15.6.2",
+    "styled-components": "^4.1.3"
```

Comment:
Shouldn't react be a peerDependency as well as a devDependency? Like you've done here: https://github.com/bbc/psammead/pull/407/files#diff-c11da1456f9036b64ebb3664660409a2R30

Reply:
yep